pkgs/development/python-modules/sfepy/default.nix

```diff
@@ -9,15 +9,19 @@
 , cython
 , python
 , sympy
+, meshio
+, mpi4py
+, psutil
+, openssh
 }:
 
 buildPythonPackage rec {
   name = "sfepy_${version}";
```

Comment:
```suggestion
  pname = "sfepy";
```

Reply:
Done. No idea why or how that was there.